### ISSUE TITLE
UniXcoder: reproduce AdvTest result

### ISSUE THREAD

**Issue by stovecat** [microsoft/CodeBERT#133]
Hi, I trained and evaluated UniXcoder on code search tasks following in README.md and got nice results (72.0 on CSN Python, 70.4 on CosQA), excepting for 40.3 MRR score on AdvTest. 

Meanwhile, I found that the score 40.3 is the same with "w/o AST" on Table 1, though CosQA result on "w/o AST" row is lower than my reproduction (reported: 68.7, reproduced: 70.4). 

So my question is, does the "UniXcoder" row result use flatten-AST on fine-tuning time?

**Comment by guoday** [comment]
No, we don't use flatten-AST on fine-tuning time. Since the  training set distribution of AdvTest is different from the test set, the result (MRR score) will be unstable (i.e. different random seed will affect performance). We try to fix random seed. However, even with a fixed random seed, different machines will have different results.
